Modules/Scripted/DataProbe/DataProbeLib/SliceViewAnnotations.py

```diff
@@ -433,6 +442,13 @@ def makeAnnotationText(self, sliceLogic):
                     self.dicomVolumeNode = 1
                 else:
                     self.dicomVolumeNode = 0
+                if sliceNode.GetSlabReconstructionEnabled():
+                    self.cornerTexts[3]['8-SlabReconstructionThickness']['text'] = 'Thickness: ' + str(sliceNode.GetSlabReconstructionThickness()) + ' mm'
```

Comment:
suffix should be obtained from the unit node.

Look into getting the suffix using code like the following:

```python
unitNode = slicer.app.applicationLogic().GetSelectionNode().GetUnitNodeID("length")
```